### DISCUSSION TITLE
linearGradient with three colors?

### DISCUSSION THREAD

**Discussion by Mubasher-Skylinx** [tomik23/circular-progress-bar#75] [Q&A]
![image](https://user-images.githubusercontent.com/116548357/217216808-115908e0-7bd8-4241-95c0-be03aef994cd.png)

**Comment by tomickigrzegorz** [comment]
Maybe a description of what's not working?
I'm not going to learn anything from this stamp-sized picture.

**Reply by Mubasher-Skylinx** [reply]
i am saying that gradient of three colors doesn't apply. its showing like the picture i sent. but it should be from one direction to the other like in a line. but its treating it as a circle starting from bottom to top.

it should be a gradient like guage meter like this

![image](https://user-images.githubusercontent.com/116548357/217450748-c43b1a77-71f2-4925-8808-938229e707b0.png)
but obviously with your style

**Reply by tomickigrzegorz** [reply]
Unfortunately, that's how lineargradient in svg works, it's a [tonal transition between colors](https://www.w3.org/TR/SVG11/pservers.html#LinearGradientElement)  it's not a bug, that's how it works.
If you want it to look like your example, you'll have to use a different library, unfortunately.
Gradient doesn't work on svg circle elements. Have a look here maybe it will help you - [svg-multiple-color-on-circle-stroke](https://stackoverflow.com/questions/18206361/svg-multiple-color-on-circle-stroke)
